src/.vuepress/components/notuptodate.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+<div class="custom-block warning" v-if='this.$page.frontmatter.lang == "fr-FR"'>
+  <p class="custom-block-title">
+    Cette page n’est pas à jour.
+  </p> 
+  <p>
+    D’importantes modifications ont été faites dans la version anglaise de cette. Nous vous recommandons d’utiliser celle-ci.
+  </p>
+</div>
+<div class="custom-block warning" v-else>
+  <p class="custom-block-title">
+    This page isn't up to date.
+  </p> 
+  <p>
+    Major changes have been made in the English version of this page. We suggest you to use this one.
```

Comment:
Still strange wording. A more direct version of what you're implying
```suggestion
    Major changes have been made in the English version of this page. We suggest you use this revision instead.
```

Though shouldn't this be something along the lines of..
```suggestion
    Major changes have been made in the English version of this page. We suggest you use the English revision until this page is up to date
```

Reply:
Thanks for the suggestion. I made the modification in English and French in 65b41a4.

Reply:
Looks great!